lib/build/build.js

```diff
@@ -152,7 +145,9 @@ class Build {
       }
     }
 
-    await updateStep('success', output)
+    await updateStep('success')
+
+    return ret
```

Comment:
Changes on L130..L150 allow step functions to actually return something useful instead of using what they return as "step output" to display in status pages. Steps now have to call their updateOutput parameter to update the output displayed on status pages.